__tests__/pages/exercises/[lessonSlug].test.js

```diff
@@ -1,12 +1,122 @@
 import React from 'react'
-import { render, waitFor } from '@testing-library/react'
+import {
+  render,
+  waitFor,
+  screen,
+  waitForElementToBeRemoved
```

Comment:
`waitForElementToBeRemoved` is not used.

Reply:
Good catch, thanks!